packages/firebase_admob/android/build.gradle

```diff
@@ -0,0 +1,36 @@
+group 'io.flutter.plugins.firebaseadmob'
+version '1.0-SNAPSHOT'
+
+buildscript {
+    repositories {
+        jcenter()
+    }
+
+    dependencies {
+        classpath 'com.android.tools.build:gradle:2.3.0'
+    }
+}
+
+allprojects {
+    repositories {
+        jcenter()
+    }
+}
+
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 25
+    buildToolsVersion '25.0.3'
+
+    defaultConfig {
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+    }
+    lintOptions {
+        disable 'InvalidPackage'
+    }
+    dependencies {
+        compile 'com.google.firebase:firebase-core:10.2.1'
```

Comment:
Update core to 11 to be consistent with other plugins. I think we also agreed on specifying the version as "11.0.+" so we don't have o release an update for every minor firebase release.